packages/dataviews/src/dataforms-layouts/panel/index.tsx

```diff
@@ -122,7 +122,7 @@ function FormField< Item >( {
 							/>
 							<field.Edit
 								key={ field.id }
-								data={ data }
+								value={ field.getValue( { item: data } ) }
```

Comment:
So, the core change here is decoupling components from data changes. That sounds good. However, what if the value is an object or array? Wouldn't we have the same issue? I'm thinking of fields that represent a list of categories, for example.

API-wise, I'm not strongly opposed to doing something like this where the derived state (value) is calculated outside the Edit — though I don't love it. Are there alternatives we can consider that don't collocate derived state away from Edit? 🤔 Does it make sense to pass a `getData` stable reference instead?

Reply:
> However, what if the value is an object or array? Wouldn't we have the same issue? I'm thinking of fields that represent a list of categories, for example.

This has no impact, the instance of that object or array won't change when you edit another field